.github/scripts/update-featured-repo.js

```
@@ -0,0 +1,224 @@
+  
+  if (GITHUB_TOKEN) {
+    headers['Authorization'] = `token ${GITHUB_TOKEN}`;
+  }
+
+  for (let attempt = 0; attempt <= retries; attempt++) {
+    try {
+      const data = await fetchData(url, headers);
+      return data;
+    } catch (error) {
+      if (error.statusCode === 403 && error.headers && error.headers['x-ratelimit-remaining'] === '0') {
+        const resetTime = parseInt(error.headers['x-ratelimit-reset']) * 1000;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)